submissions/mubarakOlaoluwa/JavaScript/PrintingProductPrice/printingproductprice.js

```diff
@@ -0,0 +1,27 @@
+var productlist = ['object1', 'object2', 'object3'];
+    
+    productlist[0] = {
+            productname1: 'Powerbank',
+            imageurl1: 'imageurl1',
+            price1: '#10,000'
+    };
+
+    productlist[1] = {
+        productname2: 'Hp laptop',
+        imageurl2: 'imageurl2',
+        price2: '#200,000'
+    };
+
+    productlist[2] = {
+        productname3: 'Mobile Phone',
+        imageurl3: 'imageurl3',
+        price3: '#120,000'
+    };
+
+    document.write (productlist[0].productname1 + ' with ' + productlist[0].imageurl1 +
```

Comment:
This is a good solution however, can we look at one that doesnt require repetition. Imagine you are printing price for over 1000 products. I mean a javascript function that is reusable

Reply:
I got it. I have corrected that and I am remaking the pull request now. Please check it out. I love your comment; it made me think wider. Great one. Thanks!!